spec/check_localization_progress_spec.rb

```diff
@@ -0,0 +1,337 @@
+require 'spec_helper.rb'
+require 'webmock/rspec'
+
+describe Fastlane::Actions::CheckTranslationProgressAction do
+  before(:each) do
+    allow(FastlaneCore::UI).to receive(:'message')
+    allow(FastlaneCore::UI).to receive(:'interactive?').and_return(true)
+  end
+
+  it 'does not fail when all the languages are above the set threshold' do
+    langs = [
+      {:lang_name => 'arabic', :lang_code => 'ar', :current => '2,087', :fuzzy => '0', :waiting => '0', :untranslated => '0'},
+      {:lang_name => 'german', :lang_code => 'de', :current => '2,078', :fuzzy => '2', :waiting => '3', :untranslated => '4'},
+      {:lang_name => 'spanish', :lang_code => 'es', :current => '2,085', :fuzzy => '0', :waiting => '0', :untranslated => '2'}
+    ]
+
+    stub = stub_request(
+      :get,
+      'https://translate.wordpress.org/projects/apps/my-test-project/dev')
+      .to_return(
+        status: 200,
+        body: generate_glotpress_response_body(languages: langs)
+      )
+
+    expect(FastlaneCore::UI).to receive(:'message').with('Done')
+
+    Fastlane::Actions::CheckTranslationProgressAction.run(
+      glotpress_url:'https://translate.wordpress.org/projects/apps/my-test-project/dev',
+      language_codes: 'ar de es'.split(),
```

Comment:
I recently learned about [Ruby's percent strings](https://ruby-doc.org/core-2.7.3/doc/syntax/literals_rdoc.html#label-Percent+Strings) which is an IMHO simpler / more idiomatic way to achieve this. 


```suggestion
      language_codes: %w[ar de es],
```